packages/integration-karma/test/component/LightningElement.disconnectedCallback/index.spec.js

```diff
@@ -157,3 +159,94 @@ it('should associate the component stack when the invocation throws', () => {
     expect(error.message).toBe('throw in disconnected');
     expect(error.wcStack).toBe('<x-disconnected-callback-throw>');
 });
+
+describe('disconnectedCallback for components with a custom render()', () => {
+    it('should invoke disconnectedCallback for children when parent is removed', () => {
+        let disconnectedCallbackInvoked = false;
+        function disconnectedCallback() {
+            disconnectedCallbackInvoked = true;
+        }
+        function tmpl($api, $cmp) {
```

Comment:
this should be imported, should not be manually created because it will break once we make changes to compiler.